src/videojs-contrib-hls.js

```diff
@@ -309,7 +345,7 @@ export default class HlsHandler extends Component {
     });
 
     this.segments = new SegmentLoader({
-      currentTime: this.tech_.currentTime.bind(this.tech_),
+      currentTime: () => this.tech_.currentTime(),
```

Comment:
`Function#bind` is an issue on pages with broken polyfills like prototypejs and mootools.